index.html

```diff
@@ -1,16 +1,117 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Great Idea - Responsive I</title>
-
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+  <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+  <div class="wrapper">
```

Comment:
You may extract the header out of this wrapper so that its padding/margin styles will not affect the header.
This will allow the borders to get to the ends of the screen.